CHANGELOG.md

```diff
@@ -1,9 +1,39 @@
 # Unreleased
 
+# 9.13.0 (Feature release)
+
+## New features
+
+### Make Sass errors clearer to users
+
+Previously, it was not obvious to users if Sass has stopped updating because of an error. An error would be printed to the Terminal, but nothing would happen in the browser. We know that this error was easy to miss, which could cause confusion.
```

Comment:
```suggestion
Previously, it was not obvious to users if Sass had stopped updating because of an error. An error would be printed to the Terminal, but nothing would happen in the browser. We know that this error was easy to miss, which could cause confusion.
```